python/pyarrow/tests/test_fs.py

```diff
@@ -15,24 +15,21 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from datetime import datetime, timezone, timedelta
 import gzip
 import os
 import pathlib
 import pickle
-
-import pytest
 import weakref
+from datetime import datetime, timedelta, timezone
```

Comment:
Ideally this would have stayed above ("datetime" is before "gzip").

Reply:
Yes, sorry, fighting a linter on my local machine. Will undo the import changes.